### PR TITLE
[CP] Katello 4.4.0.rc2 cherry-picks

### PR DESCRIPTION
**Pull request by ianballou**
#### What are the changes introduced in this pull request?
CPs for Katello 4.4.0.rc2.

#### Considerations taken when implementing this change?
There were a lot of CPs!  I had to manually edit a little bit of React code due to some conflicts, but it was all very basic.  Most of it was related to the recent linting fixes on top of some other React code that isn't making it into Katello 4.4, so mostly deletions rather than re-writing code.

#### What are the testing steps for this pull request?
1) Check that the tests pass.
2) Check that I didn't somehow bump to the wrong version
3) Check that the CHANGELOG looks okay.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,25 +19,33 @@
...
-### Content Views
- * Extend CV API to support bulk removing versions ([#34168](https://projects.theforeman.org/issues/34168), [9e6b0d07](https://github.com/Katello/katello.git/commit/9e6b0d07ef198f7a0d55399e536844cb0908150b))
-
-### Web UI
- * New Host details overview - Installable errata card ([#33083](https://projects.theforeman.org/issues/33083))
-
 ### Foreman Proxy Content
  * Provide RHSM and content URL as info providers ([#32835](https://projects.theforeman.org/issues/32835), [f3daacdb](https://github.com/Katello/katello.git/commit/f3daacdb4f2cb5d13e11401db708b41a0ed1644d))
 
 ### Other
```
Comment (chris1984) [inline CHANGELOG.md:25]: Should we move these items to a better category instead of other?

File: CHANGELOG.md [hunk 2]
```diff
@@ -55,24 +81,20 @@
...
 ### Organizations and Locations
  * Creating Organization produces two Audit logs ([#33952](https://projects.theforeman.org/issues/33952), [f83c7856](https://github.com/Katello/katello.git/commit/f83c785650bf03d7f2db00e3aabd7faac9448983), [33994b57](https://github.com/Katello/katello.git/commit/33994b57ed3e28a09ed9974fe58108f856287790))
 
-### Content Views
- * Versions API with wrong/deleted CV id shows all versions in the org ([#33860](https://projects.theforeman.org/issues/33860), [211af427](https://github.com/Katello/katello.git/commit/211af427cb65d74f8ef4d892d1d29c61a191bd84))
-
 ### Client/Agent
  * Old ApplicableContentHelper references cause `rake katello:import_applicability` to fail ([#33554](https://projects.theforeman.org/issues/33554), [15ec6dab](https://github.com/Katello/katello.git/commit/15ec6dab54333667ed1a98b3556445f2b56ce57f))
 
 ### Other
```
Comment (chris1984) [inline CHANGELOG.md:93]: Same here

**Review by chris1984 — APPROVED** [review]
ACk, one small nitpick. Up to you if you want to change it, for 4.3 I moved the stuff from other to the better categories but not a blocker if you don't want to.

**Comment by ianballou** [comment]
@chris1984 I edited the changelog

**Comment by chris1984** [comment]
APJ

**Pull request merged**